legal-api/migrations/versions/2fd104a5f3b5_.py

```diff
@@ -0,0 +1,42 @@
+"""empty message
+
+Revision ID: 2fd104a5f3b5
+Revises: e7abbdeb252a
+Create Date: 2023-06-27 07:07:08.006306
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2fd104a5f3b5'
+down_revision = 'e7abbdeb252a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('party_roles', schema=None) as batch_op:
```

Comment:
Are we deleting party_roles or renaming it?

Reply:
We'll be deleting `party_roles` and `parties` table.  `entity_roles`, `legal_entities` and `colin_entities` will be used.  

The current migration flow/scripts transfer all the existing data into the new LEAR model first which is why I'm including the `change_filing_id` here.  From there, `legal_name_updates.sql` is used to move the  party related data into the new tables(`entity_roles`, `legal_entities` and `colin_entities`).

I think once we are confident where things are we can remove the `party_roles` and `parties` table.  IThe `signing_party_id` in the `resolutions` table will also need to be removed.